Exercises/Google-Page/Index.html

```diff
@@ -3,8 +3,51 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Google Search</title>
+        <link rel="stylesheet" href="Style.css"/>
 </head>
 <body>
-    
+                <nav>
+                    <div>
+                    <a href="https://about.google">About</a>
+                    <a href="https://store.google.com/">Store</a>
+                    </div>
+                    <div class="nav-right">
+                    <a href="https://accounts.google.com/">Gmail</a>
+                    <a href="https://images.google.com/">Images</a>
+                    <img src="menu.png" height="15px" width="15px"/>
+                        <a href="https://accounts.google.com/"> 
+                                <img src="Nikhil Profile Pic.jpg" class="profile-pic">
+                        </a>
```

Comment:
Good job making "menu" image smaller and profile picture rounded 👍 
Also nice that you wrapped profile picture into `<a>` tag. 🎉
Here `<a>` tag doesn't have to be shifted as it's on same level as `<img>` tag.